README.md

```diff
@@ -1,21 +1,66 @@
 # Heat Pump Installer Chatbot Prototype
 
-This is just testing my git.
+This repository contains the **Heat Pump Installer Chatbot**, a prototype chatbot designed to assist heat pump installers with on-the-job queries via WhatsApp. Built with a focus on large language model (LLM) integration, the chatbot serves as a knowledge-enhancing tool, enabling installers to access relevant information and solutions in real time.
 
-## Setup
+## Key Features
 
-- Meet the data science cookiecutter [requirements](http://nestauk.github.io/ds-cookiecutter/quickstart), in brief:
-  - Install: `direnv` and `conda`
-- Run `make install` to configure the development environment:
-  - Setup the conda environment
-  - Configure `pre-commit`
+- **WhatsApp Integration**: Chatbot accessible directly from WhatsApp for seamless communication.
+- **Powered by LLM**: Utilises advanced language models to provide accurate and contextual responses.
+- **Installer-Focused**: Designed with the needs of heat pump installers in mind, offering targeted insights and troubleshooting support.
 
-## Contributor guidelines
+## Prerequisites
 
-[Technical and working style guidelines](https://github.com/nestauk/ds-cookiecutter/blob/master/GUIDELINES.md)
+Before setting up the project, ensure you have the following installed:
 
----
+- **Python 3.10+**
+- **Docker**
+- **poetry**
 
-<small><p>Project based on <a target="_blank" href="https://github.com/nestauk/ds-cookiecutter">Nesta's data science project template</a>
-(<a href="http://nestauk.github.io/ds-cookiecutter">Read the docs here</a>).
-</small>
+## Installation & Deployment
+
+For detailed instructions on getting the app up and running, see the [README](api/README.md) in the `api` directory
+
+## Project Structure
+
+-asf_HP_installer_chatbot/
+├── api/                         # Core chatbot application code
+├── asf_hp_installer_chatbot/    # Pipeline code
```

Comment:
so what's contained with the `asf_hp_installer_chatbot` directory is basically the proof of concept that we developed with the initial whatsapp interface, the actual code used for the Renbee chatbot is contained within api and rag (this is likely what people will want to have a look at, although the PoC might be useful if they want to start simple).